chat/views/get_last_conversation_messages.py

```diff
@@ -0,0 +1,66 @@
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from chat.models import Conversation, Message
+from chat.serializers import MessageSerializer
+from constants.header_constants import HEADER_USER_EMAIL
+
+from utils.http_utils import generate_error_response
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+from constants.profile_constants import MAX_NO_OF_MESSAGE_CONTEXT
+# Create Django Rest Endpoint that returns a list of messages for a given conversation
+
+@swagger_auto_schema(
+    method='get',
+    deprecated=True,
+    operation_description=f"Get last {MAX_NO_OF_MESSAGE_CONTEXT} messages from current user's conversation",
+    operation_id=f"Get last {MAX_NO_OF_MESSAGE_CONTEXT} messages from current user's conversation",
+    operation_summary=f"Get last {MAX_NO_OF_MESSAGE_CONTEXT} messages from current user's conversation",
+    
+    responses={
+        "200": openapi.Response(
+            description="Messages retrieved successfully",
+            examples={
+                "application/json": {
+                    "timestamp": "2021-08-30 14:00:00",
+                    "status": 200,
+                    "msg": "Messages retrieved successfully",
+                    "data": []
+                }
+            }
+        ),
+        "400": openapi.Response(
```

Comment:
In a different class this was given as 400 directly without quotations, is something missed here?

Reply:
Nope, both works